packages/components/card/src/Card.tsx

```diff
@@ -37,8 +40,29 @@ export default defineComponent({
       return children.value.some(node => node.type && (node.type as any).name === 'IxCardGrid')
     })
 
+    const accessor = useFormAccessor<boolean>('selected')
+    const isSelected = computed(() => accessor.valueRef.value ?? false)
+    const isSelectable = computed(() => props.selectable ?? false)
+    const isDisabled = computed(() => props.disabled ?? accessor.disabled.value)
+    const handleClick = () => {
+      if (isDisabled.value || !isSelectable.value) {
+        return
+      }
+      callEmit(props.onChange, !isSelected.value)
+      accessor.setValue(!isSelected.value)
+    }
```

Comment:
```suggestion
   const [isSelected, setSelected ] = useState(props.selectable)
    const handleClick = () => {
      if (props.disabled || !iprops.selectable) {
        return
      }
      const newSelected = !isSelected.value
      setSelected(newSelected)
      callEmit(props.onSelectedChange, newSelected)
    }
```

card 不是表单组件，所以这里不应该用 useFormAccessor， 直接从 props 里拿值就好了